src/CMakeLists.txt

```diff
@@ -11,23 +11,7 @@ else()
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -include config.h")
 endif()
 
-set(DISPLAYCLUSTER_SHARED_HEADERS
-  ${CMAKE_CURRENT_SOURCE_DIR}/Event.h
-  ${CMAKE_CURRENT_SOURCE_DIR}/MessageHeader.h
-  ${CMAKE_CURRENT_SOURCE_DIR}/NetworkProtocol.h
-  ${CMAKE_CURRENT_SOURCE_DIR}/PixelStreamSegment.h
-  ${CMAKE_CURRENT_SOURCE_DIR}/PixelStreamSegmentParameters.h
-)
-
-set(DISPLAYCLUSTER_SHARED_SOURCES
-  ${CMAKE_CURRENT_SOURCE_DIR}/Event.cpp
-  ${CMAKE_CURRENT_SOURCE_DIR}/log.cpp
-  ${CMAKE_CURRENT_SOURCE_DIR}/MessageHeader.cpp
-)
-
 if(BUILD_CORE_LIBRARY)
```

Comment:
We should do more changes here, CORE_LIBRARY should always be build now that the stream library is separated